gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/oneconvergence_service_node_driver.py

```diff
@@ -1062,6 +1072,10 @@ def _fetch_template_and_params(self, context, update=False):
                                   'param': stack_params})
         return (stack_template, stack_params)
 
+    def _get_service_profile_type(self, context):
+        #return context.current_node['description']  # description should contain ASAV or VYOS
+        return SERVICE_PROFILE_TYPES[0]     # For now return 'ASAV'
```

Comment:
We conveyed to sungard that service vendor or service flavor field will have the vendor type and HA. Not the  description field.
Why are we returning ASAV for now?